msprime/trees.py

```diff
@@ -1339,18 +1359,24 @@ def load_text(nodes, edges, sites=None, mutations=None, sequence_length=0, stric
         not supplied or zero this will be inferred from the set of edges.
     :param bool strict: If True, require strict tab delimiting (default). If
         False, a relaxed whitespace splitting algorithm is used.
+    :param string encoding: Encoding used for text representation.
+    :param bool base64_metadata: If True, metadata is encoded using Base64
+        encoding; otherwise, as plain text.
     :return: The tree sequence object containing the information
         stored in the specified file paths.
     :rtype: :class:`msprime.TreeSequence`
     """
-    node_table = parse_nodes(nodes, strict=strict)
+    node_table = parse_nodes(nodes, strict=strict, encoding=encoding,
```

Comment:
Nothing essential, but I do wince a bit every time I see code visually indented like this. My objection is that it makes it harder than it needs to be to change code --- if we decided to change the ``node_table`` variable to ``nodes``, say, then you'd also have to reindent. Or, if we changed the name of ``parse_nodes``, we couldn't just do a global search and replace, we'd have to reindent all the visual indents as well.

Anyway, it doesn't really matter, so feel free to ignore. I just had to get that one off my chest!